web-app/HeimScripts/heatmap/run.r

```diff
@@ -7,6 +7,8 @@ main <- function(max_rows=50){
   if(ncol(df) > 3){
     variances <- apply(df[,3:ncol(df)],1,var) # Calculating variance per probe
     df["SIGNIFICANCE"] <- variances
+    df["MEAN"] <- apply(df[,3:ncol(df)],1,mean, na.rm = T)
+    df["SD"] <- apply(df[,3:ncol(df)],1,sd, na.rm = T)
```

Comment:
The purpose of calculating SD is not ordering - it is to calculate Z-score I put it there just for convenience and later delete.